haystack/modeling/conversion/transformers.py

```diff
@@ -4,7 +4,7 @@
 import torch
 from transformers import AutoModelForQuestionAnswering
 
-from haystack.modeling.model import adaptive_model as am
+from haystack.modeling.model.adaptive_model import AdaptiveModel
```

Comment:
I think this might also result in a circular import.

Reply:
Now that I look at the code, this part is also quite odd. Why do we have a class with three static methods that could simply be part of `AdaptiveModel`? Is there something I miss? I'm going to look for other usages of `Converter` and if there aren't I'm going to try merging this class into `AdaptiveModel`.

Reply:
Seems reasonable.